pkg/features/httproute.go

```diff
@@ -100,6 +100,9 @@ const (
 
 	// This option indicates support for the name field in the HTTPRouteRule (extended conformance)
 	SupportHTTPRouteNamedRouteRule FeatureName = "HTTPRouteNamedRouteRule"
+
+	// This option indicates support for the cors filter in the HTTPRouteFilter (extended conformance)
+	SupportHTTPRouteCORS FeatureName = "HTTPRouteNamedRouteRule"
```

Comment:
is this the expected feature name? Or is it something like 
```suggestion
	SupportHTTPRouteCORS FeatureName = "HTTPRouteCors"
```
?

Reply:
will fix